arm-web/2015-08-01/swagger/AppServiceCertificateOrders.json

```diff
@@ -821,47 +821,36 @@
     "AppServiceCertificate": {
```

Comment:
1. It seems we renamed 'AppServiceCertificate' to 'AppServiceCertificateResource', any specific reason for the new name?
2. 'AppServiceCertificate' model is not used anywhere right? if so please remove it.